web/urls.py

```diff
@@ -40,34 +40,13 @@
         views.ExperimentAssetsProxyView.as_view(),
         name="experiment-assets-proxy",
     ),
-    path("faq/", flatpages_views.flatpage, dict(url="/faq/"), name="faq"),
-    path(
-        "scientists/",
-        flatpages_views.flatpage,
-        dict(url="/scientists/"),
-        name="scientists",
-    ),
-    path(
-        "resources/",
-        flatpages_views.flatpage,
-        dict(url="/resources/"),
-        name="resources",
-    ),
-    path(
-        "contact_us/",
-        flatpages_views.flatpage,
-        dict(url="/contact_us/"),
-        name="contact_us",
-    ),
-    re_path(
-        r"^(?P<path>assets/.*)$",
-        views.ExperimentAssetsProxyView.as_view(),
-        name="experiment-assets-proxy",
-    ),
-    re_path(
-        r"^(?P<path>fonts/.*)$",
-        views.ExperimentAssetsProxyView.as_view(),
-        name="experiment-fonts-proxy",
-    ),
-    path("", flatpages_views.flatpage, dict(url=""), name="home"),
+    path("", views.HomeView.as_view(), name="home"),
+    path("faq/", views.FAQView.as_view(), name="faq"),
+    path("privacy/", views.PrivacyView.as_view(), name="privacy"),
+    path("scientists/", views.ScientistsView.as_view(), name="scientists"),
+    path("contact_us/", views.ContactView.as_view(), name="contact"),
+
+# Remaining flat pages
+    path("resources/", flatpages_views.flatpage , dict(url="/en-us/resources/"), name="resources"),
+    path("termsofuse/", flatpages_views.flatpage , dict(url="/en-us/termsofuse/"), name="termsofuser"),
```

Comment:
Would it be too much of a lift to port over the remaining flatpages to templates?  If so, let me know and I can on that work.

Reply:
No problem, I can do this! We just didn't do them because we don't plan to translate them - the resources because they're very country specific, and the terms-of-use because we're not lawyers.

Something on my mind: do you think we should get rid of the "flatpages" directory for the templates. This is going to hang around like an old legacy, long after it is relevant.

Reply:
Thanks for do this.  Yes, I do think we should get rid of the directory, after they've been all moved to templates.

Reply:
Done!